bundles/org.openhab.persistence.rrd4j/src/main/java/org/openhab/persistence/rrd4j/internal/RRD4jPersistenceService.java

```diff
@@ -152,7 +152,12 @@ public synchronized void store(final Item item, @Nullable final String alias) {
         }
         final String name = alias == null ? item.getName() : alias;
 
-        RrdDb db = getDB(name);
+        RrdDb db = null;
+        try {
+            db = getDB(name);
+        } catch (Exception e) {
+            logger.warn("Failed to open rrd4j database '{}' ({})", name, e.getClass().getName());
```

Comment:
Do we want to log also that item could not be found, or just return immediately when we can't even open the database? Alternatively, item name could be included in this log entry to avoid having two log entries.
```suggestion
            logger.warn("Failed to open rrd4j database '{}' ({})", name, e.getClass().getName());
            return List.of();
```

Reply:
thins function is void, so not necessary here

Reply:
Sorry, you are right of course, went too fast.